common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -258,7 +258,7 @@ const SearchTabs: FunctionComponent<Props> = ({
             }}
             ariaDescribedBy="images-form-description"
             isImageSearch={true}
-            shouldShowFilters={isEnhanced && shouldShowFilters} // non js images filters doesnt work hide for now\
+            shouldShowFilters={shouldShowFilters} // non js images filters doesnt work hide for now\
```

Comment:
What does "for now" mean in this case (should we create a ticket/this be a TODO comment?)

Reply:
I thought this comment was relating to the `isEnhanced` variable, and if Adi has got images filters work without JavaScript, can we delete it?